src/Microsoft.Identity.Web.Diagnostics/OsHelper.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Web.Diagnostics
+{
+    internal class OsHelper
```

Comment:
Copied from MSAL.NET; Not sure this is the right place for it, but it seems to be the only "common" DLL.

Reply:
This was pulled from R9, which is now in .NET 8, but not available on older frameworks, so leaving it in place for now.